src/configurations/sources/shopify/ui-config.json

```diff
@@ -3,11 +3,36 @@
     {
       "title": "Connection Settings",
       "fields": [
+        {
+          "type": "singleSelect",
+          "label": "Version of app",
+          "value": "version",
+          "options": [
+            {
+              "name": "Client side tracking with the Rudderstack tracker script (Legacy).",
+              "value": "v1"
+            },
+            {
+              "name": "Client side tracking with the Rudderstack web pixel (Recommended).",
+              "value": "v2"
+            }
+          ],
+          "defaultOption": {
+            "name": "tracker",
+            "value": "v1"
+          },
+          "footerNote": "Please note that RudderStack app is required to be installed in your Shopify store for the legacy version. For the recommended version, you need to install the RudderStack Pixel on your Shopify store.",
```

Comment:
we have setup guide just at the top right in the page - so should be good with just the text here